{"uuid": "f6459ad3-0b72-49e9-9bb2-b4f74c26dadf", "counter": 7, "stdout": "\u001b[1;31mfatal: [192.168.1.1]: UNREACHABLE! => {\"changed\": false, \"msg\": \"Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused\", \"unreachable\": true}\u001b[0m", "start_line": 5, "end_line": 6, "runner_ident": "69abc1fc-acaf-4621-b500-103c0d4e84bc", "event": "runner_on_unreachable", "pid": 19354, "created": "2022-04-28T17:11:09.209162", "parent_uuid": "9e73ed77-7fb2-88a2-ce91-00000000000f", "event_data": {"playbook": "deploy.yaml", "playbook_uuid": "b8109558-3e18-4418-8020-af8cbe91fafa", "play": "this is a  hello world example", "play_uuid": "9e73ed77-7fb2-88a2-ce91-000000000007", "play_pattern": "all", "task": "Gathering Facts", "task_uuid": "9e73ed77-7fb2-88a2-ce91-00000000000f", "task_action": "gather_facts", "task_args": "", "task_path": "/Users/devops/data/project/python/pgoops/upload/iac/helloworld/project/deploy.yaml:2", "host": "192.168.1.1", "remote_addr": "192.168.1.1", "start": "2022-04-28T17:11:09.187502", "end": "2022-04-28T17:11:09.209074", "duration": 0.021572, "res": {"unreachable": true, "msg": "Failed to connect to the host via ssh: ssh: connect to host 192.168.1.1 port 22: Connection refused", "changed": false}, "uuid": "f6459ad3-0b72-49e9-9bb2-b4f74c26dadf"}}